### PR TITLE
feature/header

### PR DESCRIPTION
**Pull request by Nazar865**

### CODE REVIEW

**Comment by Nazar865** [comment]
on this commit is so long history, I turned off it for future

File: Corzbank.Web/ClientApp/src/app/header/header.component.html [hunk 1]
```diff
@@ -1 +1,22 @@
-<p>header works!</p>
\ No newline at end of file
+<div class="navbar">
+    <div class="left-block">
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.html]: left/right block looks weird and not informative. Think of better naming

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    letter-spacing: 1px;
+    width: 80%;
+    margin: 10px 0;
+}
+.left-block , .right-block{
+    display: flex;
+    justify-content: space-between;
+    width: 24%;
+    align-items: baseline;
+    align-items: center;
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.scss]: all styles below overwrite styles from top, that means line 22 overwrite line 21, remove from 21

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    font-size: 25px;
+    letter-spacing: 1px;
+    width: 80%;
+    margin: 10px 0;
+}
+.left-block , .right-block{
+    display: flex;
+    justify-content: space-between;
+    width: 24%;
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.scss]: in percentages?? change to pixels

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+}
+.navbar{
+    // background-color: green;
+    height: 100px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 25px;
+    letter-spacing: 1px;
+    width: 80%;
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.scss:13]: pixels

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    display: flex;
+    width: 140px;
+    justify-content: space-between;
+    align-items: center;
+}
+.right-block .account{
+    display: -webkit-inline-box;
+    width: 140px;
+    height: 40px;
+    -webkit-box-align: center;
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.scss]: why do you need that style?

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    transform: scale(1.3);
+}
+.left-block .services{
+    display: flex;
+    width: 140px;
+    justify-content: space-between;
+    align-items: center;
+}
+.right-block .account{
+    display: -webkit-inline-box;
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.scss]: -webkit?? change in all places

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    width: 140px;
+    height: 40px;
+    -webkit-box-align: center;
+    border-radius: 32px;
+    background: transparent;
+    border: 3px solid #77D353;
+    padding: 0 10px;
+    -webkit-box-pack: center;
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.scss]: same as line 47

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+    font-size: 18px;
+
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/app/header/header.component.scss]: odd line

File: Corzbank.Web/ClientApp/src/styles.css [hunk 9]
```diff
@@ -14,3 +14,13 @@ code {
   background-color: #1b6ec2;
   border-color: #1861ac;
 }
+body{
+  display: block;
+  text-align: -webkit-center;
+  background-color: #24203A;
+}
+.main{
```
Comment (ghost) [inline Corzbank.Web/ClientApp/src/styles.css]: what is main selector? Is it used somewhere?

**Review by ghost — COMMENTED** [review]
photos folder should be renamed to images

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 4 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss:13]: image
in this case it's ok

File: Corzbank.Web/ClientApp/src/app/header/header.component.html (hunk 1 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.html]: I haven't idea about how to rename right-block, I searhed some sites and they write left/right-side, A/b-block and smth like this

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 2 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss]: resolved

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 3 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss]: resolved

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 5 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss]: resolved

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 6 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss]: resolved, the only one is in  body style.css

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 7 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss]: resolved

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 8 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss]: resolved

File: Corzbank.Web/ClientApp/src/styles.css (hunk 9 above)
Comment (Nazar865) [reply Corzbank.Web/ClientApp/src/styles.css]: resolved

File: Corzbank.Web/ClientApp/src/app/header/header.component.html (hunk 1 above)
Comment (ghost) [reply Corzbank.Web/ClientApp/src/app/header/header.component.html]: if some sites have that wrong implementation, it doesn't mean it is correct and you should follow their logic

File: Corzbank.Web/ClientApp/src/app/header/header.component.scss (hunk 4 above)
Comment (ghost) [reply Corzbank.Web/ClientApp/src/app/header/header.component.scss:13]: in header yeah, make sense

File: Corzbank.Web/ClientApp/src/app/header/header.component.html (hunk 1 above)
Comment (ghost) [reply Corzbank.Web/ClientApp/src/app/header/header.component.html]: there is no left-block section at all, not sure why you created that selector in styles
right-block could be splitted into exchange and account, without a wrapper

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Approved, but resolve those minor comments

**Pull request merged**